src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -49,6 +56,9 @@ public FindCommand parse(String args) throws ParseException {
             predicates.add(getBlockPredicate(argMultimap.getValue(PREFIX_BLOCK).get()));
         }
 
+        parseStudentGroupsForFind(argMultimap.getAllValues(PREFIX_STUDENT_GROUP))
+            .ifPresent(studentGroupSet -> predicates.add(new StudentGroupPredicate(studentGroupSet)));
+
```

Comment:
Is there a reason you did not follow the previous standard of 
`if (argMultimap.getValue(PREFIX_STUDENT_GROUP).isPresent()) {
      predicates.add(getStudentGroupPredicate(argMultimap.getValue(PREFIX_STUDENT_GROUP).get()));
 }`?

Reply:
Not really, updated the pr to follow the convention. Thanks.